pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -255,8 +263,18 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 				// NLBs require a different health check interval than CLBs
 				service.Annotations[awsLBHealthCheckIntervalAnnotation] = awsLBHealthCheckIntervalNLB
 			} else {
+				var additionalTags []string
+				if platformSpec != nil && platformSpec.AWS != nil {
+					for _, userTag := range platformSpec.AWS.UserTags {
+						if len(userTag.Key) > 0 {
+							additionalTags = append(additionalTags, userTag.Key+"="+userTag.Value)
+						}
+					}
+				}
+				service.Annotations[awsLBAdditionalResourceTags] = strings.Join(additionalTags, ",")
```

Comment:
Doesn't the annotation work for NLBs as well?  Looking at the cloud provider code, I thought it did.

Reply:
I need to check. I just wanted something end-2-end at the moment to flesh out unknowns.

Reply:
NLBs work; updated.